wait-for-deployment

```diff
@@ -67,7 +68,7 @@ echo "Waiting for deployment with timeout ${timeout} seconds"
 monitor_timeout $$ &
 readonly timeout_monitor_pid=$!
 
-trap 'kill ${timeout_monitor_pid}' EXIT #Stop timeout monitor
+trap 'kill -- -${timeout_monitor_pid}' EXIT #Stop timeout monitor
```

Comment:
the `--` will stop processing arguments as options so that `-${timeout_monitor_pid}` won't be recognized as a signal. Negative pids are interpreted as process group ids.